packages/kbn-config-schema/src/types/maybe_type.ts

```diff
@@ -14,7 +14,7 @@ export class MaybeType<V> extends Type<V | undefined> {
       type
         .getSchema()
         .optional()
-        .default(() => undefined, 'undefined')
+        .default(() => undefined)
```

Comment:
No longer allowed to define a value description when setting a default value. We weren't using that anywhere anyway.